addons/laserpointer/XEH_preInit.sqf

```diff
@@ -6,4 +6,11 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+["visionMode", {
```

Comment:
`if ((currentVisionMode player) == 2) exitWith {};` //0.0048
`if (visionMode == 2) exitWith {};` //0.0042 
There you go. Moving the variable outside doesn't do much